common/log.hpp

```diff
@@ -74,7 +74,7 @@ enum class LogLevel {
 
 using ExpectedLogLevel = expected::Expected<LogLevel, error::Error>;
 
-inline string to_string_level(LogLevel lvl) {
+inline string ToStringLogLevel(LogLevel lvl) {
```

Comment:
Doesn't matter now, but this commit should be either a `chore` or a `style` category. This probably would not be a new patch release ;)

Reply:
You don't like it's **fixing** the ugly name you gave it? :stuck_out_tongue_winking_eye:

Reply:
:laughing: